MediaBrowser.Api/Playback/MediaInfoService.cs

```diff
@@ -520,10 +520,7 @@ private async Task<PlaybackInfoResponse> GetPlaybackInfo(Guid id, Guid userId, s
                         streamInfo.StartPositionTicks = startTimeTicks;
                         mediaSource.TranscodingUrl = streamInfo.ToUrl("-", auth.Token).TrimStart('-');
                         mediaSource.TranscodingUrl += "&allowVideoStreamCopy=false";
-                        if (!allowAudioStreamCopy)
-                        {
-                            mediaSource.TranscodingUrl += "&allowAudioStreamCopy=false";
-                        }
+                        mediaSource.TranscodingUrl += "&allowAudioStreamCopy=false";
```

Comment:
I have no idea what this option does ahah but why not update the calling code to set `allowAudioStreamCopy` to false instead of removing this?

Reply:
I'm quite the amateur coder, I'm not sure what the calling code is 😅.

So if you enable Force LiveTV Transcoding in a user's profile, ForceRemoteSourceTranscoding becomes true. It then forces the logic for the container, video. Audio if necessary. I'm having issues where the audio isn't transcoding and it causes the audio to be out of sync. I would like to be able to make this only remux as an option as well but I kept getting inconsistencies with quality. I was hoping to leave the audio as optional but I'm finding new issues there as well.

Reply:
Yeah okay that makes sense, thanks for the explanation!

I also realized my original question doesn't make sense, I thought `allowAudioStreamCopy` wasn't being used anywhere else in this function for some reason